### ISSUE TITLE
Generating decorations is slow for interactive GUI

### ISSUE THREAD

**Issue by aymanhab** [opensim-org/opensim-core#895]
Latest API, calling from a test program on gait2392 model takes more than 800 ms to generateDecorations (2 yr old windows machine) Can't be interactive at that speed.

**Comment by aymanhab** [comment]
@chrisdembia I'm adding a test case that outputs the times but need to use a model, what's the latest mechanism so I don't duplicate models inside the Java testing folder? Thanks

**Comment by chrisdembia** [comment]
Unfortunately, the macro I made for this won't work as-is (https://github.com/opensim-org/opensim-core/blob/master/cmake/OpenSimMacros.cmake#L147). For now, perhaps just use something like the following in Bindings/Java/tests/CMakeLists.txt:

```
file(COPY "${OPENSIM_SHARED_TEST_FILES_DIR}/<filename>"
                 DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/<name-of-test>")
```

where <name-of-test> might be "TestBasics"

**Comment by aymanhab** [comment]
This has been fixed by PR #903 Closing.
